doc/refman/RefMan-syn.tex

```diff
@@ -925,38 +936,40 @@ \subsection{Interpretation scopes used in the standard library of {\Coq}}
 
 \subsubsection{\tt type\_scope}
 
-This includes infix {\tt *} for product types and infix {\tt +} for
-sum types. It is delimited by key {\tt type}.
+This scope includes infix {\tt *} for product types and infix {\tt +} for
+sum types. It is delimited by key {\tt type}, and bound to the class 
+{\tt Sortclass}, which is used as an interpretation scope for certain
+types within an {\tt Ltac} goal match.
 
 \subsubsection{\tt nat\_scope}
 
-This includes the standard arithmetical operators and relations on
+This scope includes the standard arithmetical operators and relations on
 type {\tt nat}. Positive numerals in this scope are mapped to their
 canonical representent built from {\tt O} and {\tt S}. The scope is
-delimited by key {\tt nat}.
+delimited by key {\tt nat}, and bound to the class {\tt nat}.
```

Comment:
Maybe worth to write "coercion class" rather than "class", or otherwise "class" with a reference to the coercion class section, or otherwise "type". This is to avoid the confusion with "type classes", which is the default meaning for "class" it seems. (More generally, I'm not very satisfied with having reused for scopes the class concept of the coercion system. This could eventually be made better.)

Similarly at other places.